engine/action_contact.go

```diff
@@ -16,16 +17,14 @@ func (c *Client) UpsertContact(ctx context.Context, ctcFName, ctcPaymail, reques
 		return nil, err
 	}
 
-	pmSrvnt := &PaymailServant{
-		cs: c.Cachestore(),
-		pc: c.PaymailClient(),
-	}
-	contactPm, err := pmSrvnt.GetSanitizedPaymail(ctcPaymail)
+	paymailService := c.PaymailService()
+
+	contactPm, err := paymailService.GetSanitizedPaymail(ctcPaymail)
```

Comment:
If the the `Pm` stands for `PayMail` - it should be `contactPM` (according to golang styleguide)
- But I would prefer `contactPaymail`

Reply:
and what if it stand for `Paymail` not `PayMail`?